src/components/Navigation.tsx

```diff
@@ -1,5 +1,84 @@
+"use client";
+import Image from "next/image";
+import Link from "next/link";
+import React, { useState } from "react";
+import logo from "@/public/logo.webp";
+import { items } from "@/data/navigation";
+import { IoMdMenu } from "react-icons/io";
+
 const Navigation = () => {
-  return <div className="fixed w-screen">Navigation</div>;
+  const [selected, setSelected] = useState("");
+  const [nav, setNav] = useState(false);
+  const handleNav = () => {
+    setNav(!nav);
+  };
+  return (
+    <div className="sticky top-0 z-30 ml-10 mr-10 flex items-center justify-between border-b-4 border-hearts-blue bg-hearts-beige text-2xl">
```

Comment:
Make sure for any margin, padding and width, use percent or view width. Example:  m-[10%] or w-[20vw].
For any text in your component, only use view width. Example: text-[5vw].

This will improve the responsiveness for the website on all screen, meaning that the ratio will always be the same no matter how big your screen is.